geopandas/geodataframe.py

```diff
@@ -747,6 +747,40 @@ def astype(self, dtype, copy=True, errors="raise", **kwargs):
         # do not return a GeoDataFrame
         return pd.DataFrame(df)
 
+    def to_postgis(
+        self, con, table, if_exists="fail", schema=None, dtype=None, index=False
+    ):
+
+        """
+        Upload GeoDataFrame into PostGIS database.
+
+        Parameters
+        ----------
+        con : sqlalchemy.engine.Engine
+            Connection.
+        table : str
+            Target table to write the data.
+        if_exists : str
+            What to do if table exists already: 'replace' | 'append' | 'fail'.
```

Comment:
You can copy the description from pandas as it is more explicit. May be the case for other as well.